ledger-core/virtual/integration/method_test.go

```diff
@@ -1305,7 +1305,6 @@ func Test_MethodCall_HappyPath(t *testing.T) {
 		server, ctx := utils.NewUninitializedServer(nil, t)
```

Comment:
defer server.stop()

Reply:
can't do this because it will placed in loop.  server.stop() called in end of loop manually

Reply:
done